### PR TITLE
List and detail payments

### PR DESCRIPTION
**Pull request by amber-marichi**
added payment app with its model, serializer, views and urls for list and detail endpoints

### CODE REVIEW

File: payment/views.py [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+    def get_queryset(self):
+        queryset = self.queryset
+        if not self.request.user.is_staff:
+            return queryset.filter(borrowing__user=self.request.user)
+        return queryset.all()
+
+
+class PaymentDetailView(generics.RetrieveAPIView):
+    queryset = Payment.objects.select_related("borrowing__user").all()
```
Comment (NazarHladaniuk) [inline payment/views.py:22]: as i remember you can remove .all here

**Review by NazarHladaniuk — CHANGES_REQUESTED** [review]

File: payment/views.py (hunk 1 above)
Comment (yura-hudzovskyi) [reply payment/views.py:22]: Yeah, you can remove .all()

**Review by yura-hudzovskyi — CHANGES_REQUESTED** [review]

File: payment/views.py (hunk 1 above)
Comment (amber-marichi) [reply payment/views.py]: removed, ty

**Review by yura-hudzovskyi — APPROVED** [review]
LGTM!

**Review by NazarHladaniuk — APPROVED** [review]

**Pull request merged**